src/styling/ProjectsPage.css

```diff
@@ -1,101 +1,174 @@
 /** Multiple card view **/
 
-.pg-container {
+.projects-page {
   max-width: 1400px;
+  margin: 3rem auto;
+}
+
+.projects-page .flow-images {
+  margin: 1rem 0;
 }
 
 .preview-card {
-  border-radius: 10px;
-  margin: 20px 0px;
-  padding: 5px;
+  border-radius: 16px;
+  margin: 16px 0px;
+  box-shadow: 8px 8px 0px 2px rgba(249,245,227,1);
+  padding: 4px;
+  cursor: pointer;
+  background-color: var(--main-grey);
 }
 
-.urgent-tag {
-  border-radius: 50px;
+.preview-card:hover {
   background-color: var(--main-yellow);
-  font-weight: bold;
   color: var(--main-black);
-  border: transparent;
 }
 
```

Comment:
Hm not a huge fan of the current on-hover state, maybe we could change the card-border from white to grey as well? @Raquellevy for additional ideas
<img width="465" alt="Screen Shot 2020-07-16 at 7 57 16 AM" src="https://user-images.githubusercontent.com/43453743/87668659-7b5dd900-c73a-11ea-96bc-97c5982e04e5.png">

Reply:
Don't know if it's just me, but the blue for urgent tag and especially More details... is a little hard to read on hover. Could we change to a slightly darker blue on-hover maybe?

Reply:
Hey @liu-caro ! tbh I kind of like the yellow with the cream border. Especially since it's just a hover state it's not suuppperr important to be legible, but open to other ideas